### PR TITLE
[New] Added sider to profile page

### PR DESCRIPTION
**Pull request by Rizvida1**
- There is now a sider anchor on profile page

### CODE REVIEW

File: services/frontend-v3/src/i18n/en_CA.json [hunk 1]
```diff
@@ -136,6 +136,7 @@
   "profile.acting.period.start.date": "Acting start date",
   "profile.address": "Address",
   "profile.branch": "Branch",
+  "profile.basic": "Basic Info",
```
Comment (s-glmxmd) [inline services/frontend-v3/src/i18n/en_CA.json]: Where is this used? If its for the main card I don't think we need a title for the card.

File: services/frontend-v3/src/i18n/fr_CA.json [hunk 2]
```diff
@@ -139,6 +139,7 @@
   "profile.acting.period.start.date": "Date de début intérimaire",
   "profile.address": "Adresse",
   "profile.branch": "Branche",
+  "profile.basic": "Info de base",
```
Comment (s-glmxmd) [inline services/frontend-v3/src/i18n/fr_CA.json]: Same comment for english... not sure why where this is used or why we would need one.

File: services/frontend-v3/src/i18n/en_CA.json (hunk 1 above)
Comment (s-glmxmd) [reply services/frontend-v3/src/i18n/en_CA.json]: Would recommend Personal Information or Employee Information

File: services/frontend-v3/src/i18n/fr_CA.json (hunk 2 above)
Comment (s-glmxmd) [reply services/frontend-v3/src/i18n/fr_CA.json]: Translation would be Information personnel or Information d'employée

File: services/frontend-v3/src/i18n/en_CA.json (hunk 1 above)
Comment (Rizvida1) [reply services/frontend-v3/src/i18n/en_CA.json]: It's used for the anchor thing I made

**Review by s-glmxmd — APPROVED** [review]
Good job Rizvi! 😄

**Pull request merged**